template-parts/home-asset-map.php

```diff
@@ -0,0 +1,13 @@
+<?php if( get_field("asset_map_custom_field")): ?>
+  <div class="container home-asset-map bg-dark asset-map-banner-bg-image" style="background-image: url('<?php echo get_field('asset_map_image')['url'] ?>')">
+    <div class="row">
+      <div class="col p-0">
+        <div class="col-5 bg-white p-4">
```

Comment:
Have you checked to see what this would look like on smaller screens? `col-5` may be too small

Reply:
Good catch. Something like `col-12 col-md-5` would work better.